recipes-wpe/wpewebkit/wpewebkit_0.1.bb

```diff
@@ -39,7 +39,7 @@ PROVISIONING_x86 = ""
 PROVISIONING_hikey-32 = ""
 PROVISIONING_dragonboard-410c-32 = ""
 
-PACKAGECONFIG ?= "2dcanvas deviceorientation fullscreenapi encryptedmediav1 fetchapi gamepad geolocation indexeddb libinput logs mediasource notifications nativevideo sampling-profiler shadowdom subtitle subtlecrypto udev video webaudio ${WPE_PLATFORM}"
+PACKAGECONFIG ?= "2dcanvas deviceorientation fullscreenapi encryptedmediav1 fetchapi gamepad geolocation indexeddb libinput logs mediasource notifications nativevideo sampling-profiler shadowdom subtitle subtlecrypto touch udev video webaudio ${WPE_PLATFORM}"
```

Comment:
Fair point! I'll straighten those out as well. It indeed started as a showcase thing, however in the longer run I think its better to be a little conservative and have people enable extra features like those you mention locally.